var/spack/repos/builtin/packages/python/package.py

```diff
@@ -199,6 +199,9 @@ class Python(AutotoolsPackage):
     # https://github.com/python/cpython/pull/16717
     patch('intel-3.6.7.patch', when='@3.6.7:3.6.8,3.7.1:3.7.5 %intel')
 
+    # Fixes RPATH issue with Intel compilers
+    patch('python-intel-rpath.patch', when='@3.1:3.99 %intel')
```

Comment:
No need for an upper bound here, it will likely change once your upstream contribution gets merged anyway. Do we want a second patch for Python 2 like we have for other compilers?

Reply:
What is Spack's preference? I think Python 2 is no longer supported so we've been taking the approach of not paying attention to it. But we have seen in other PRs that you have recommended we add the dep. We don't need Python 2, but can add it if you like.

Reply:
> What is Spack's preference? 

In this particular instance, I'll leave it up to you. I'm sure there are people out there trying to build Python 2 with Intel who would appreciate a patch, although I don't use Python 2 anymore.

> I think Python 2 is no longer supported so we've been taking the approach of not paying attention to it.

Python 2 is no longer supported, although many Spack users still use it (unfortunately).

> But we have seen in other PRs that you have recommended we add the dep.

I think you're talking about dependencies like `depends_on('python@2.7:2.8,3.5:')`. This kind of dependency allows someone to use Python 2 even if it is no longer supported. We don't want to prevent people from using Python 2, although encouraging them to switch to Python 3 is fine.

Reply:
Ok thanks - we will attempt to make an analogous Python 2 patch.